stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -315,9 +297,13 @@ public class MessageListView : ConstraintLayout {
         }
     }
 
-    private fun initUnseenMessagesButton() {
-        binding.scrollToBottomButton.setOnClickListener {
-            binding.chatMessagesRV.scrollToPosition(lastPosition())
+    private fun initScrollHelper() {
+        scrollHelper = MessageListScrollHelper(
+            recyclerView = binding.chatMessagesRV,
+            scrollButtonView = binding.scrollToBottomButton,
+            scrollToBottomButtonEnabled = scrollToBottomButtonEnabled,
```

Comment:
The `MessageListView` property we're setting here is set in `configureAttributes`, which is invoked later than `initScrollHelper`. I think this will result in us always passing in the default `true` here.

Then again, just changing the order of these is dangerous, because we can easily change it back again later without noticing. Perhaps we should make the `MessageListView` property a `Delegates.notNull` value (as it can't be `lateinit`, being a primitive).

Reply:
You mean `scrollToBottomButtonEnabled` property?

Reply:
@zsmb13 Can you check the latest commit? I ensured that we crash if the ordering is broken